DevelopmentProposals/readme.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+This directory contains all the development proposals for OpenSim.
+
+## How to create a proposal
+
+1. Create a new directory under directory ***DevelopmentProposals***.
```

Comment:
I don't have a strong opinion, either. The thought behind OpenSim/ was to obscure from users. I don't think this will be a huge problem since the folder is called "Proposals". I'd say let's leave in the root unless anyone has strong objections.